repo/modules.go

```diff
@@ -61,7 +64,11 @@ func importRepoRulesModules(filename string, _ *RemoteCache) (repos []Repo, err
 		if mod.Main {
 			continue
 		}
-		pathToModule[mod.Path] = mod
+		if mod.Replace != nil {
```

Comment:
Check that `mod.Replace.Path` is not a file path. Both `path/filepath.IsAbs` and `go/build.IsLocalImport` should return false.

If it is a local path, warn that `go_repository` doesn't support file path replacements and skip the module.

Reply:
LMK how that looks. I wasn't sure what a good warning message would look like